packages/spotlight/src/container.js

```diff
@@ -1101,6 +1102,22 @@ function notifyEnterContainer (direction, previous, previousContainerIds, curren
 	});
 }
 
+/**
+ * Returns a closest container that wrap the element and has positionTargetOnDescendantsFocus configured
```

Comment:
It looks that this function finds the furthest container, not the closest container. And, in my opinion, the current behavior makes sense. Let's discuss about it. @juwonjeong

Reply:
I fixed to find the closest container. Thank you